src/test/resources/features/CreatePosition.feature

```diff
@@ -0,0 +1,10 @@
+Feature: CreatePosition
+
+  Scenario: EO-10 Verify that a position cannot be created without required fields
+    Given 'SSI' page is loaded
+    And User is authenticated
+    And I click on "Estructura Organizacional" menu button
```

Comment:
dont use "I"

Reply:
Example

According to the test case it should be:
Given I loging to 'SMARTHOUSE' page with properly credentials
When go to 'Estructura Organizacional' on 'Header' page
And navigate to 'Gerencia General' option on 'Dashboar/Left Panel/etc...' page
And click over 'Eliminar' button on 'Genencia General' page
Then 'Area Gerencia General' title should be displayed